src/lib/tabs/tab-group.scss

```diff
@@ -32,19 +32,28 @@ md-ink-bar {
 .md-tab-body-wrapper {
   position: relative;
   overflow: hidden;
-  flex-grow: 1;
   display: flex;
+  transition: height 0.5s $ease-in-out-curve-function;
 }
 
 // Wraps each tab body 
-.md-tab-body {
-  display: none;
-  overflow: auto;
-  box-sizing: border-box;
-  flex-grow: 1;
-  flex-shrink: 1;
-  &.md-tab-active {
-    display: block;
+md-tab-body {
+  display: block;
+  position: absolute;
```

Comment:
Right, there should probably be `md-fill` mixin or something like that with `position: absolute`. That mixin does not work right now.